packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -109,6 +104,17 @@ const AccountSwitcher = observer(({ history, is_mobile, is_visible }) => {
         }
     };
 
+    const { oAuthLogout } = useOauth2({ handleLogout: logoutHandler });
+
+    const handleLogout = async () => {
+        closeAccountsDialog();
+        if (is_positions_drawer_on) {
+            togglePositionsDrawer(); // TODO: hide drawer inside logout, once it is a mobx action
```

Comment:
[Q] same question here, why are we toggling first then logout?

Reply:
Same as above, but in this case we want to close the drawer first.